### PR TITLE
chore: Move drun ownership to the languages team

### PR DESCRIPTION
**Pull request by dsarlis**
`drun` is only used by the Languages team for testing. Moving the ownership to them so they can independently evolve it as they see fit.

### CODE REVIEW

**Review by basvandijk — APPROVED** [review]
I just added the dfinity/languages team as a collaborator to this repo which means the CODEOWNER file is valid again.

Also consider adding the languages team to [.github/workflows/team-channels.json](https://github.com/dfinity/ic/blob/master/.github/workflows/team-channels.json) so they get notified on a slack channel whenever they're requested to review.

**Comment by dsarlis** [comment]
> Also consider adding the languages team to [.github/workflows/team-channels.json](https://github.com/dfinity/ic/blob/master/.github/workflows/team-channels.json) so they get notified on a slack channel whenever they're requested to review.

I'll leave this up to the languages team.

**Pull request merged**
